src/style.css

```diff
@@ -0,0 +1,45 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.stars_row {
+  display: flex;
+  height: 100%;
+  width: 100%;
```

Comment:
height and width are reduntant for html block element
height and width by default are 100%

<img width="783" alt="Screenshot 2022-03-21 at 17 09 30" src="https://user-images.githubusercontent.com/37452763/159291029-4bb33770-73ee-4bec-839e-797289a932ea.png">